dev/com.ibm.ws.security.spnego/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -29,10 +29,10 @@ canonicalHostName=Use canonical host name
 canonicalHostName.desc= Controls whether you want to use the canonical host name.
 
 krb5Config=The Kerberos configuration file with full path
-krb5Config.desc=Specifies the fully qualified Kerberos configuration path and name. Standard variable substitutions, such as ${server.config.dir}, can be used when specifying the directory path.
+krb5Config.desc=Deprecated: use configFile attribute on the <kerberos> element instead. Specifies the fully qualified Kerberos configuration path and name. Standard variable substitutions, such as ${server.config.dir}, can be used when specifying the directory path.
 
 krb5Keytab=The Kerberos keytab file name and include path
-krb5Keytab.desc=Specifies the fully qualified Kerberos keytab  path and name. Standard variable substitutions, such as ${server.config.dir}, can be used when specifying the directory path. The Kerberos keytab file contains a list of keys that are analogous to user passwords. It is important for hosts to protect their Kerberos keytab files by storing them on the local disk.
+krb5Keytab.desc=Deprecated: use keytab attribute on the <kerberos> element instead. Specifies the fully qualified Kerberos keytab  path and name. Standard variable substitutions, such as ${server.config.dir}, can be used when specifying the directory path. The Kerberos keytab file contains a list of keys that are analogous to user passwords. It is important for hosts to protect their Kerberos keytab files by storing them on the local disk.
```

Comment:
Specifies the fully qualified Kerberos keytab  path and name.

remove extra space between "keytab" and "path"

**change**
Standard variable substitutions, such as ${server.config.dir}, can be used when specifying the directory path.
**to**
Standard variable substitutions, such as ${server.config.dir}, can be used when you specify the directory path.